vyper/ast/signatures/interface.py

```diff
@@ -99,8 +99,20 @@ def extract_sigs(sig_code, interface_name=None):
         interface_ast = [
             i
             for i in vy_ast.parse_to_ast(sig_code["code"], contract_name=interface_name)
-            if isinstance(i, vy_ast.FunctionDef)
-            or isinstance(i, vy_ast.EventDef)
+            # all the nodes visited by ModuleNodeVisitor.
+            if isinstance(
+                i,
+                (
+                    vy_ast.FunctionDef,
+                    vy_ast.EventDef,
```

Comment:
I'm not sure if Events should be importable, typically they are not interface-able, but I could be convinced (e.g. `Approval` for `ERC20`)

Reply:
I don't think this really means the Event is importable, it just forces the compiler to register them. I think it's good for overall correctness (imagine if in the future a function signature somehow depends on an Event being present, without parsing Events in this routine there will be a bug in that future).

Reply:
I suppose this holds for `implements: ERC20`

Reply:
yes, checking correctness of an interface re: events is why it was added i believe